frontend/src/components/mine/Summary/MineSummary.js

```diff
@@ -70,8 +70,15 @@ const renderSummaryTSF = (tsf, partyRelationships) => (
   </Col>
 );
 
+// Since end date is stored at yyyy-mm-dd, comparing current Date() to
+// the the start of the next day ensures appointments ending today are displayed.
+const getTomorrow = (end_date) => {
+  const dayAfterEndDate = new Date(end_date);
+  return dayAfterEndDate.setDate(dayAfterEndDate.getDate() + 1);
+};
+
 const isActive = (pr) =>
-  (!pr.end_date || Date.parse(pr.end_date) >= new Date()) &&
+  (!pr.end_date || getTomorrow(pr.end_date) > new Date()) &&
```

Comment:
Instead of using this custom function, we can likely leverage the existing `momentjs` library that we're using for something like: `moment(x.end_date).add(1, 'days') > new Date()`

Thoughts?

Docs:
https://momentjs.com/docs/#/parsing/string/
https://momentjs.com/docs/#/manipulating/add/

Reply:
I agree 👍 I was looking at this before lunch and was going to comment that getTomorrow seemed awkward.

Reply:
implemented with your suggestions